Gemfile

```diff
@@ -63,10 +64,6 @@ gem 'foodsoft_discourse', path: 'plugins/discourse'
 #gem 'foodsoft_documents', path: 'plugins/documents'
 
 
-group :production do
-  gem 'exception_notification'
-end
```

Comment:
Why?

Reply:
i want to use `ExceptionNotifier.notify_exception` on some places, where the admin should be notified about exceptions. E.g. when receiving a mail throws an exception or when a specific action in a controller has a `rescue`, but the error might be relevant to admins too

Reply:
That's a good idea! :+1: